de.fu_berlin.inf.dpp.intellij/src/de/fu_berlin/inf/dpp/core/ui/util/CollaborationUtils.java

```diff
@@ -74,6 +78,8 @@ private CollaborationUtils() {
     public static void startSession(List<IResource> resources,
         final List<JID> contacts) {
 
+        projectAPI.saveAllDocuments();
```

Comment:
This should not be necessary.

Reply:
You are right, this is not needed. My bad. Must have misread my test results.